# Main parameters
number_of_robots: 2

